behaviour/graql/language/delete.feature

```diff
@@ -233,7 +306,10 @@ Feature: Graql Delete Query
     Then the integrity is validated
 
 
-  Scenario: delete a role player from a relation removes the player from the relation
+  Scenario: delete an attribute ownership using too-specific (downcasting) type throws
```

Comment:
i think this one is worth implementing, is an interesting edge case

Reply:
Bah, looks like I missed this one AND a second one.
Moving this one down the file (to after "Scenario: an attribute ownership can be deleted using its supertype as a label")
And implementing "Scenario: delete a role player from a relation using a super-role removes the player from the relation"

Reply:
Added